source/macos/AdaptiveCards/AdaptiveCards/Views/ACRView.swift

```diff
@@ -118,7 +127,11 @@ class ACRView: ACRColumnView {
         repeat {
             if let handlers = parentView?.inputHandlers {
                 for handler in handlers {
-                    guard handler.isValid else { continue }
+                    if renderConfig.supportsSchemeV1_3 && !handler.isValid {
+                        handler.showError()
+                        canSubmit = false
+                        continue
+                    }
                     dict[handler.key] = handler.value
```

Comment:
Why was guard removed?
doesn't it do the same thing with it?

Reply:
With Guard the code was looking a bit confusing to me, since line 135 goes into the else part, which is the actual logic to be implemented in the loop (generally we put the actual logic below guard, right?). The three lines in the if block would go below the else

Reply:
But if you flip the condition to !supports!_3 && isValid
thenn flow will remain how you want